components/Tabs/examples/supplemental/TabBarIconExampleSupplemental.m

```diff
@@ -234,8 +234,8 @@ - (void)addStarCentered:(BOOL)centered {
   [self.starPage addSubview:starView];
   [starView sizeToFit];
 
-  CGFloat x = centered ? 1 : (arc4random_uniform(199) + 1) / 100;  // 0 < x <=2
-  CGFloat y = centered ? 1 : (arc4random_uniform(199) + 1) / 100;  // 0 < y <=2
+  CGFloat x = centered ? 1 : (CGFloat)((arc4random_uniform(199) + 1) / 100.0);  // 0 < x <=2
+  CGFloat y = centered ? 1 : (CGFloat)((arc4random_uniform(199) + 1) / 100.0);  // 0 < y <=2
```

Comment:
I notice you're using this value between 0 and 2 as the multiplier and 0 as the constant with NSLayoutAttributeCenterX and NSLayoutAttributeCenterY. I understand how the multiplier is used for proportional constraints, but I don't understand its use right here. How does this value being 1 or 2 make a difference, and why is it random?

Reply:
Here's an example:

```
Bounds = {0, 0, 100, 100}
x = {0.25}
y = {1.33}

Star position = {superview.center.x * 0.25, superview.center.y * 1.33}
Star position = {0.5 * 0.25, 0.5 * 1.33}
Star position = {0.125, .665}
```

Reply:
It's random so the stars don't overlap? I guess the alternative would be a "grid" of stars, but I don't think there's any particular reason to choose one over the other.

Reply:
Huh okay, i didn't realize you could do that with the center attributes. And what are these stars for again? I just don't know

Reply:
They're stars that appear in the example.

![Simulator Screen Shot - iPhone 7 - 2019-04-03 at 10 30 42](https://user-images.githubusercontent.com/1753199/55487198-a121a600-55fb-11e9-8363-e0ae27114b44.png)

Reply:
Oh I see haha. Cool.